code/drasil-docLang/Drasil/DocumentLanguage/Core.hs

```diff
@@ -96,7 +96,7 @@ data IntroSec = IntroProg Sentence Sentence [IntroSub]
 
 -- | Introduction subsections
 data IntroSub where
-  IPurpose :: Sentence -> IntroSub
+  IPurpose :: [Sentence] -> IntroSub
```

Comment:
Note that you should consider using `NonEmpty` (which is a non-empty list), as we don't want to ever deal with length 0 lists. That can be a future improvement.